<launch>
  <arg name="uuv_name"/>
  <arg name="filename" default="$(find-pkg-share uuv_control_utils)/config/example_waypoints.yaml"/>
  <arg name="start_time" default="-1"/>
  <arg name="interpolator" default="lipb"/>

  <group>
    <push-ros-namespace namespace="$(var uuv_name)"/>
    <node pkg="uuv_control_utils"
          exec="send_waypoint_file.py"
          name="send_waypoint_file"
          output="screen">
      <param name="filename" value="$(var filename)"/>
      <param name="start_time" value="$(var start_time)"/>
      <param name="interpolator" value="$(var interpolator)"/>
    </node>
  </group>

</launch>
